libs/frontend/feature-build/src/components/toolbar/ToolbarText.vue

```diff
@@ -271,13 +241,16 @@ watch(
 }
 
 .font-size {
-  width: 72px;
+  width: 76px;
   margin-left: 4px;
   :deep(.ps-input) {
     border: 1px solid $color-light1;
     background-color: rgba(255, 255, 255, 0.8);
     font-size: 15px;
     padding-right: 32px;
   }
+  :deep(.ps-input-suffix) {
+    user-select: none;
+  }
```

Comment:
I add `user-select: none` to the suffix of font-size input to implicitly inform users they (currently) can't change the unit of font-size in the toolbar.

Before:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/5a7db745-99ca-45aa-b34e-3148b1aef03b

After:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/787d589a-6386-464b-9371-2f803f79fd33

I was thinking if we should use a dropdown of font-size units to replace the suffix text so that users can change not only the size but the unit at the same time, but I think that'll take some time; maybe we can come back here when most of the bugs are fixed! Also, I'm not sure if we should allow users to change the unit if font-size is actually a resolved value of a theme variable (e.g. `${size-h1}`).

I hope this change can temporarily let users giving up the idea of changing font-size unit in the toolbar 😬.

Reply:
Works for me! I made a note to improve font-size unit handling later.